### ISSUE TITLE
Client startup causes high CPU usage

### ISSUE THREAD

**Issue by metablaster** [OpenSpace/OpenSpace#2307]
I downloaded a fresh copy of v0.18, run the installer and launched the program, then my CPU jumped to 100% workload, vents keep spinning like crazy and so it continued indefinitely, I was forced to close down opnespace because it feels like my CPU would burn out.
I was able to use openspace, ex. zooming etc., it seems but CPU load never got below 100%

I expect the program to launch and use mx 5-20% of CPU.

I also assume this could be due to compile of shaders in the background for first time use, but unless you can confirm this is the case and how long it would take for shaders to compile I'm not in the mood to burn my CPU if it's a bug.

**Comment by alexanderbock** [comment]
:wave:  What kind of CPU do you have?  Unless you are choosing a configuration that VSync enabled or you create one that does (see bottom of the attached picture), then OpenSpace will always try to run full-bore until either one core of the CPU or the GPU is at 100%.  Particularly in the beginning when the layers for a planet load in, getting close to 100% is not unreasonable, but depending on your internet connection to should drop down close to (100/n)% where `n` is the number of threads your CPU has.  What framerate are you getting when it is at 100%?  

Also to try out, if you create a configuration with VSync enabled (start Launcher, click "New" beneath the dropdown menu of "Window Options", check the checkbox next to "Enable VSync" and then "Apply without Saving" and start OpenSpace), do you still get full CPU usage?

Unless you have it ultra-overclocked, there is no need to worry about a burn out, modern CPUs will automatically reduce the voltage so that they don't go over 100°, so you are safe that way :)

We're not compiling any shaders in the background, they are all compiled as you start up OpenSpace every time.

![image](https://user-images.githubusercontent.com/4921724/201165924-720c9cd7-9bcf-4988-9549-ba97fcfa695e.png)

**Comment by metablaster** [comment]
Thanks for helpful reply, you said:

`Particularly in the beginning when the layers for a planet load in, getting close to 100% is not unreasonable, but depending on your internet connection to should drop down close to (100/n)% where n is the number of threads your CPU has.`

I took a look into console output and it said it failed to connect to network to download data, this is because of my firewall.
As soon as I opened outbound port to download data it all worked just fine, CPU works normally @33% and there was no overload.

Thanks!
